### DISCUSSION TITLE
working with 2D images of 5 channels!

### DISCUSSION THREAD

**Discussion by NouraAAR** [MIC-DKFZ/nnUNet#1958] [Q&A]
Hi everyone,

I am trying to implement nn-Unet for my project where I use 2D images with 5 channels each, corresponding to 5 energy bins. The files are in DICOM format. I was wondering if this can be used here or if any adjustments are necessary.
Thank you.

Additionally, I would like to ask if there are any online step-by-step tutorials to follow that would help in using nn-Unet?

**Comment by ancestor-mithril** [comment]
Yes, it's possible to use 5 channels. The documentation can be found here: https://github.com/MIC-DKFZ/nnUNet/tree/master/documentation. 
Some relevant links: https://github.com/MIC-DKFZ/nnUNet/blob/master/documentation/dataset_format.md and  https://github.com/MIC-DKFZ/nnUNet/blob/master/documentation/how_to_use_nnunet.md.
